pdr_backend/cli/cli_arguments.py

```diff
@@ -656,7 +658,7 @@ def __init__(self, description: str, command_name: str):
     "do_multisim": MultisimArgParser("Run >1 simulations", "multisim"),
     "do_deployer": DeployerArgPaser(),
     "do_lake": LakeArgParser("Run the lake tool", "lake"),
-    "do_analytics": AnalyticsArgParser("Run the analytics tool", "analytics"),
```

Comment:
I don't know what this command was going and why we removed it, but If we remove it we might want to remove everything related to it such as: `AnalyticsArgParser` definition and maybe other places where this is mentioned

Reply:
do_analytics -> do_ohlcv

they are the same command